tests/test-ray-tracing-common.cpp

```diff
@@ -0,0 +1,69 @@
+#include "test-ray-tracing-common.h"
+
+namespace rhi::testing {
+
+Result loadShaderPrograms(
```

Comment:
Is there a similar function in `testing.h` that could be used? Seems like this is not necessarily raytracing test specific.